src/routes/quickstarts/nuxt/setup.mdx

```diff
@@ -0,0 +1,103 @@
+# Setup a Nuxt app with Mirage for Development
+
+Mock your Nuxt app's networking code so you can develop your Nuxt app without any backend services.
+
+> This is a quickstart guide for people familiar with Mirage. If you're brand new to Mirage, take a look at the [Overview](/docs/getting-started/overview).
+
+## Step 1: Install Mirage
+
+First, make sure you have Mirage installed:
+
+```bash
+# Using npm
+npm install --save-dev miragejs
+
+# Using Yarn
+yarn add --dev miragejs
+```
+
+## Step 2: Define your server
+
+Under the plugins Nuxt folder, create a new `mirage.js` file and define your mock server.
+
+Here's a basic example:
+
+```js
+// src/plugins/mirage/mirage.js
+import { Server, Model } from "miragejs"
+
+export function makeServer({ environment = "development" } = {}) {
+  let server = new Server({
+    environment,
+
+    models: {
+      user: Model,
+    },
+
+    seeds(server) {
+      server.create("user", { name: "Bob" })
+      server.create("user", { name: "Alice" })
+    },
+
+    routes() {
+      this.namespace = "api"
+
+      this.get("/users", schema => {
+        return schema.users.all()
+      })
+    },
+  })
+
+  return server
+}
+```
+
+## Step 3: Use your server only in development
+
+Open your Nuxt app's config file (`src/nuxt.config.js`), include your `mirage.js` in plugins array if in development mode:
+
+```js{4-6}
+// src/nuxt.config.js
+export default {
+  ...
+  plugins: [
+    ...(process.env.NODE_ENV !== 'production' ? ['~/plugins/mirage/mirage.js'] : []),
+  ],
+  ...
+}
+```
+
+Now, whenever your application makes a network request in development, Mirage will intercept that request and respond with the data from your server definition.
+
+For example, given the server definition above, the following component would fetch the users `Bob` and `Alice` that we defined in `seeds`:
+
+```html
+<!-- src/index.vue -->
+<template>
+  <ul id="users">
+    <li v-for="user in users" v-bind:key="user.id">{{ user.name }}</li>
+  </ul>
+</template>
+
+<script>
+  export default {
+    name: "app",
+
+    data() {
+      return {
+        users: [],
+      }
+    },
+
+    created() {
```

Comment:
I'd use `beforeMount` or `mounted` here to signal it only works on client-side
```suggestion
    mounted () {
```

Reply:
Thanks! Sorry that this has gone stale – I think we should try to move it forward